helm/coredns-app/templates/deployment-masters.yaml

```diff
@@ -105,3 +106,4 @@ spec:
         # We need to create the /tmp folder to avoid CoreDNS crash during api-server is down
```

Comment:
How about the `Corefile` CM above? Does kubeadm `coredns` have the same volume?

Reply:
Yeah, the kubeadm has the `/etc/coredns` but it doesn't have this temp volume. I'm not sure exactly what it's for and how it helps avoid a crash, do you know?

Reply:
No idea TBH